### PR TITLE
feature: Account page redesign

### PR DESCRIPTION
**Pull request by bhongy**

### CODE REVIEW

File: apps/web/components/NewPipelineInScreen.tsx [hunk 1]
```diff
@@ -1,202 +0,0 @@
-import {useAtom} from 'jotai'
```
Comment (bhongy) [inline apps/web/components/NewPipelineInScreen.tsx:1]: This is unused. Delete to allow us to replace the old `Tabs` with the ones updated with the current design.

File: apps/web/components/api-access/PrimaryTabs.tsx [hunk 2]
```diff
@@ -1,20 +0,0 @@
-import {TabsPrimitive as Tabs} from '@usevenice/ui'
-
-interface PrimaryTabsProps<K> {
-  options: Array<{key: K; label: string}>
-}
-
-export function PrimaryTabs<K extends string>(props: PrimaryTabsProps<K>) {
```
Comment (bhongy) [inline apps/web/components/api-access/PrimaryTabs.tsx:7]: This becomes `TabsTriggers` via `@usevenice/ui`

File: apps/web/components/api-access/VeniceDatabaseExplorer/QueryOutput/ActionsBar.tsx [hunk 3]
```diff
@@ -81,22 +81,3 @@ function ExecuteQueryActionGroup(props: ExecuteQueryActionGroupProps) {
     </div>
   )
 }
-
-interface OutputFormatTabsProps {
-  options: Array<{key: OutputTabsKey; label: string}>
-}
-
-function OutputFormatTabs(props: OutputFormatTabsProps) {
```
Comment (bhongy) [inline apps/web/components/api-access/VeniceDatabaseExplorer/QueryOutput/ActionsBar.tsx:89]: Duplicate

File: apps/web/components/connections/AddSourceDialog.tsx [hunk 4]
```diff
@@ -1,16 +0,0 @@
-import {DialogPrimitive as Dialog} from '@usevenice/ui'
```
Comment (bhongy) [inline apps/web/components/connections/AddSourceDialog.tsx:1]: See https://github.com/useVenice/venice/pull/34/files#r1109440335

File: apps/web/components/connections/AddSourceCard.tsx [hunk 5]
```diff
@@ -1,24 +0,0 @@
-import {DialogPrimitive as Dialog} from '@usevenice/ui'
```
Comment (bhongy) [inline apps/web/components/connections/AddSourceCard.tsx:1]: See https://github.com/useVenice/venice/pull/34/files#r1109440335

File: apps/web/pages/account/Security.tsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+export function Security() {
+  const {register, handleSubmit, formState} = useForm<SecurityFormData>({
+    reValidateMode: 'onBlur',
+  })
+  return (
+    <form
+      className="flex flex-col items-start gap-6"
+      onSubmit={handleSubmit(async (data) => {
+        await new Promise((resolve) => setTimeout(resolve, 2000))
+        console.log('Successfully update password', data)
```
Comment (bhongy) [inline apps/web/pages/account/Security.tsx:20]: @tonyxiao I think we need an API for this. Seems like supabase only has reset password via Email flow. https://supabase.com/docs/reference/javascript/auth-resetpasswordforemail

@iwasrobbed should we change to that flow instead?

Comment (iwasrobbed) [reply apps/web/pages/account/Security.tsx:20]: Yes, please change to whatever is easiest (& we can even punt on this one for now and just ask people to logout if they need to reset pw for some reason)

**Review by iwasrobbed — APPROVED** [review]
Quick win 🚀

**Pull request merged**